schema/deploy/trigger_functions/check_for_immutable_status.sql

```diff
@@ -0,0 +1,20 @@
+-- Deploy ggircs-portal:trigger_functions/check_for_immutable_status to pg
+-- requires: schema_ggircs_portal_private
+
+begin;
+
+create or replace function ggircs_portal_private.check_for_immutable_status()
```

Comment:
The name of the function could be more precise. It's not obvious that it's related to application revisions